nyxx.interactions/lib/interactions.dart

```diff
@@ -0,0 +1,28 @@
+library nyxx_interactions;
+
+import "dart:async";
+
+import "package:logging/logging.dart";
+import "package:nyxx/nyxx.dart";
+
+// Root
+part "src/Interactions.dart";
+
+// Models
+part "src/Models/SlashCommand.dart";
```

Comment:
Please rename all folder to lowerCamelCase to be compliant with dart style

Reply:
done!